### PR TITLE
F 433 add poign art donations to our db

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #433

### CODE REVIEW

File: src/analytics/segmentQueue.ts [hunk 1]
```diff
@@ -26,10 +26,13 @@ export const processSendSegmentEventsJobs = () => {
     numberOfVerifyDonationConcurrentJob,
     async (job, done) => {
       const { event, analyticsUserId, properties, anonymousId } = job.data;
-      analytics.track(event, analyticsUserId, properties, anonymousId);
-      // When sending events to segment it misses some events when sending concurrent events
-      // So I added a queue to send them consequentially and with 1.5 second time between them
-      await sleep(1500);
+      if (event) {
+        analytics.track(event, analyticsUserId, properties, anonymousId);
```
Comment (mohammadranjbarz) [inline src/analytics/segmentQueue.ts:30]: There were some bunch of logs in our console from segment that were saying event cant be undefined, so I put this condition

File: src/server/bootstrap.ts [hunk 2]
```diff
@@ -279,10 +281,9 @@ export async function bootstrap() {
     if ((config.get('GIVING_BLOCKS_SERVICE_ACTIVE') as string) === 'true') {
       runGivingBlocksProjectSynchronization();
     }
-    await getCsvAirdropTransactions(
-      '0x0c452a7c116adb6162390f342cee84175f34e3c1bc0015e6f82773a54ace3061',
-      100,
-    );
```
Comment (mohammadranjbarz) [inline src/server/bootstrap.ts:285]: It was for test when implementing rainbow roll donations, but I forgot to delete it that time

File: src/services/poignArt/syncPoignArtDonationCronJob.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  unchainProject: Project,
+) => {
+  const ethPrice = (
+    await fetchGivHistoricPrice(poignArtWithdrawal.txHash, NETWORK_IDS.MAIN_NET)
+  ).givPriceInEth;
+  const donation = Donation.create({
+    projectId: unchainProject.id,
+    toWalletAddress: unchainProject.walletAddress,
+    isProjectVerified: unchainProject.verified,
+    fromWalletAddress: 'I should ask it from Lauren',
```
Comment (mohammadranjbarz) [inline src/services/poignArt/syncPoignArtDonationCronJob.ts]: Untill this time is not clear what address should we set for poign art donations

**Comment by mohammadranjbarz** [comment]
Hey @CarlosQ96  
This PR is ready but we need  to know what should we add for `fromWalletAddress` of donations and we should get mainnet subgraph from PoignArt guysx as I 'm off next week can you go forward this PR and staying in the loop?
I will add you to Discrod group if next week they were ready please merge this PR and needed parameters in config

( the last checkbox of #433 is related to giveback-calculation project and we dont need to do anything for that in impact-graph)

File: src/server/bootstrap.ts [hunk 4]
```diff
@@ -37,6 +37,8 @@ import { runUpdateTraceableProjectsTotalDonations } from '../services/cronJobs/s
 import { getCsvAirdropTransactions } from '../services/transactionService';
 import { runNotifyMissingDonationsCronJob } from '../services/cronJobs/notifyDonationsWithSegment';
 import { errorMessages } from '../utils/errorMessages';
+import { getPoignArtWithdrawals } from '../services/poignArt/api';
```
Comment (CarlosQ96) [inline src/server/bootstrap.ts]: Removing this from this file as it's not being used.

**Review by CarlosQ96 — APPROVED** [review]
LGTM!!! Thanks for clarifying the env vars.

**Pull request merged**